### PR TITLE
GitHub Sync: Fix GithubReader

### PR DESCRIPTION
**Pull request by val-fed**
Set `repoId`, `file`,  `type` only once, not to override these values for each comment.

### CODE REVIEW

File: tools/reviewer/job/sync/GithubReader.java [hunk 1]
```diff
@@ -195,32 +195,31 @@ private Thread getCodeThreadByFilenameAndPosition(
...
+                    .build())
+            .setType(Thread.Type.CODE);
+
     positionComments.forEach(
         comment ->
             thread
-                // TODO: Set `repoId`, `file`, `lineNumber`, `type` only once, not to override these
-                // values for each comment.
-                .setRepoId(pr.getRepoName())
                 .setCommitId(setThreadCommitId(pr, comment))
```
[the remark this answers is not in the document]
Comment (val-fed) [reply tools/reviewer/job/sync/GithubReader.java]: You are right. Fixed it.

File: tools/reviewer/job/sync/GithubReader.java [hunk 2]
```diff
@@ -195,53 +195,57 @@ private Thread getCodeThreadByFilenameAndPosition(
                 .filter(comment -> comment.getPosition() == position)
                 .collect(Collectors.toList()));
 
-    Thread.Builder thread = Thread.newBuilder();
+    GithubComment firstComment;
+    if (positionComments.isEmpty()) {
+      throw new RuntimeException("Can not find comments for a position: " + position);
```
Comment (oferb) [inline tools/reviewer/job/sync/GithubReader.java]: s/Can not/Can't

File: tools/reviewer/job/sync/GithubReader.java [hunk 3]
```diff
@@ -195,53 +195,57 @@ private Thread getCodeThreadByFilenameAndPosition(
...
+    } else {
+      firstComment = positionComments.get(0);
```
Comment (oferb) [inline tools/reviewer/job/sync/GithubReader.java]: You don't need the else - you can just;
`GithubComment firstComment = positionComments.get(0);`

File: tools/reviewer/job/sync/GithubReader.java [hunk 4]
```diff
@@ -195,53 +195,57 @@ private Thread getCodeThreadByFilenameAndPosition(
                 .filter(comment -> comment.getPosition() == position)
                 .collect(Collectors.toList()));
 
-    Thread.Builder thread = Thread.newBuilder();
+    GithubComment firstComment;
```
Comment (oferb) [inline tools/reviewer/job/sync/GithubReader.java]: Please add a comment for why you use the first comment - that you are trying to fit several comments into a thread, and you assume all comments have the same values for these fields (is that really true?)

File: tools/reviewer/job/sync/GithubReader.java (hunk 3 above)
Comment (val-fed) [reply tools/reviewer/job/sync/GithubReader.java]: Done.

File: tools/reviewer/job/sync/GithubReader.java (hunk 4 above)
Comment (val-fed) [reply tools/reviewer/job/sync/GithubReader.java]: Done.

File: tools/reviewer/job/sync/GithubReader.java (hunk 2 above)
Comment (val-fed) [reply tools/reviewer/job/sync/GithubReader.java]: Done.

**Pull request closed without merging**